From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Mon, 24 Apr 2023 15:52:50 +0200
Subject: [PATCH] Engine: Remove malloc.h

This seems to be a non-standard header. The API used by the game already
finds its way through the inclusion of other headers, so leave this out.
---
 SamTFE/Sources/Engine/Engine.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/SamTFE/Sources/Engine/Engine.h b/SamTFE/Sources/Engine/Engine.h
index 7466327d68cb3de4aa86b2f5c0ec67881bf67eb8..fe62f26e60ce33ea31a63cef619097c913dad05f 100644
--- a/SamTFE/Sources/Engine/Engine.h
+++ b/SamTFE/Sources/Engine/Engine.h
@@ -31,7 +31,6 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #endif
 
 #include <stdlib.h>
-#include <malloc.h>
 #include <stdarg.h>
 #include <stdio.h>
 #include <string.h>
